cmd/router/main_test.go

```diff
@@ -82,13 +82,14 @@ func TestSimpleModelChainer(t *testing.T) {
 		"Authorization": {"Bearer Token"},
 	}
 
-	res, err := routeStep("root", graphSpec, jsonBytes, headers)
+	res, statusCode, err := routeStep("root", graphSpec, jsonBytes, headers)
 	var response map[string]interface{}
 	err = json.Unmarshal(res, &response)
 	expectedResponse := map[string]interface{}{
 		"predictions": "2",
 	}
 	fmt.Printf("final response:%v\n", response)
+	fmt.Println(statusCode)
 	assert.Equal(t, expectedResponse, response)
```

Comment:
Can combine to the previous logs

Reply:
Not needed. Was added for debugging. Removing it.